python/pyarrow/scalar.pxi

```diff
@@ -1027,6 +1027,32 @@ cdef class ExtensionScalar(Scalar):
         return pyarrow_wrap_scalar(<shared_ptr[CScalar]> sp_scalar)
 
 
+cdef class FixedShapeTensorScalar(ExtensionScalar):
+    """
+    Concrete class for fixed shape tensor extension scalar.
+    """
+
+    def to_numpy_ndarray(self):
```

Comment:
Why not `to_numpy`?

Reply:
See above.

Reply:
Well, the `Scalar` base class doesn't have a `to_numpy` method...